events/datadog.go

```diff
@@ -70,12 +68,10 @@ func (d *DataDog) NewEvent(title string, text string, fields map[string]string,
 	if err != nil {
 		return err
 	}
-	// TODO: fix?
-	// defer g.Conn.Close()
-	err = g.Event(title, text, fields, tags)
-	if err != nil {
-		return fmt.Errorf("Error reporting Godspeed event %s: %s", title, err)
-	}
+	defer g.Godspeed.Conn.Close()
```

Comment:
Do you need to close the connection after every send? I would imagine the connection should only be closed at the end of program execution.